libs/http2-manager/http2-manager.cabal

```diff
@@ -0,0 +1,78 @@
+cabal-version:      1.12
+name:               http2-manager
+version:            0.0.1
+synopsis:           Managed connection pool for HTTP2
+description:        Helps reuse HTTP2 connections
+category:           Web
+author:             Wire Swiss GmbH
+maintainer:         Wire Swiss GmbH <backend@wire.com>
+copyright:          (c) 2020 Wire Swiss GmbH
+license:            AGPL-3
+license-file:       LICENSE
+build-type:         Simple
+
+-- cabal-fmt: glob-files test/resources/*.pem
+extra-source-files:
+  test/resources/localhost-key.pem
+  test/resources/localhost.example.com-key.pem
+  test/resources/localhost.example.com.pem
+  test/resources/localhost.pem
+  test/resources/unit-ca.pem
+
+library
+  -- cabal-fmt: expand src
+  exposed-modules:
+    HTTP2.Client.Manager
+    HTTP2.Client.Manager.Internal
+
+  other-modules:    Paths_http2_manager
+  hs-source-dirs:   src
+  ghc-options:
+    -O2 -Wall -Wincomplete-uni-patterns -Wincomplete-record-updates
+    -Wpartial-fields -fwarn-tabs -optP-Wno-nonportable-include-path
```

Comment:
what is `-optP-Wno-nonportable-include-path` for?

Reply:
I copy pasted it without thinking much :sweat_smile:

Reply:
Removed.